app/src/main/java/org/p2p/wallet/striga/offramp/ui/StrigaOffRampPresenter.kt

```diff
@@ -60,6 +66,10 @@ class StrigaOffRampPresenter(
     private val cannotClickAllAmount: Boolean
         get() = isErrorHappened
 
+    init {
```

Comment:
i would not want to put it in constructor invokation
maybe attach would be a better place?